### PR TITLE
Platform 2852

### PR DESCRIPTION
**Pull request by denisekgosnell**
Initial Pass at adding wording around the three different components of the /identity product. 

Also,  initial pass of the available documentation for /identity/match

### CODE REVIEW

File: source/includes/_identity.md [hunk 1]
```diff
@@ -167,17 +167,21 @@ curl -s -H "Authorization: Bearer $ACCESS_TOKEN" "https://platform.pokitdok.com/
...
+PokitDok's Identity Management (IdM) API queries an EMPI (Enterprise Master Patient Index) and/or MPI (Master Patient Index), both typically components of an EMR or EHR system, to find a patient identifier and details in the target EMR/EHR system. This helps providers identify the patient through past visits or other records within other EMR/EHR systems. 
 
-Available Identity endpoints:
+Within the Identity Management product, there are three availalbe interfaces: Identity, Identity History and Identity Match. The Identity API provides the foundational POST/PUT/GET capability for respectively inserting, updating and retrieving identities in the system. The Identity system updates and returns a single best record while simultaneously tracking the historical updates made to the entity. The Identity History API provides access to the historical versions of an identity. Lastly, the Identity Match API supports historical data loads for configurable duplicate entity detection.
```
Comment (dixonwh) [inline source/includes/_identity.md]: Great summary! I would recommend reworking the Identity Match API statement to indicate, in the appropriate data science manner, supports executing a configurable and tuneable match job across a data populating. The "historical data loads" portion of the statement may be a bit confusing.

File: source/includes/_identity.md [hunk 2]
```diff
@@ -239,7 +243,22 @@ External id search is executed using the "id" parameter:
...
+Learn more about our (Identity Management workflows.)[https://platform.pokitdok.com/workflows#anchor-idm]
+
 | Endpoint   | HTTP Method | Description                                                            |
 |:-----------|:------------|:-----------------------------------------------------------------------|
 | /identity/{uuid}/history | GET | Returns an identity record's change history including the insert date and historical version id|
 | /identity/{uuid}/history/{version id} | GET | Returns a historical identity record | 
+
+##### Identity Match
+
+The Identity Match API supports historical data loads for configurable duplicate entity detection. This endpoint operates asynchronously.  There are three components of the API which can be adjusted to find the best set of matches across your historical data: the match algorithm, the search fields and the match weight. The different match algorithms can be set for exact or approximate string matching. The use of different source and search fields allows for the detection of transposition within an entity's values. Lastly, setting different match weights allows you to control the importance of any individual field level match.
```
Comment (dixonwh) [inline source/includes/_identity.md]: "historical data loads"

File: source/includes/_identity.md (hunk 1 above)
Comment (denisekgosnell) [inline source/includes/_identity.md]: @dixonwh  "Lastly, the Identity Match API supports the execution of a tune-able and configurable match job which detects duplicates across a historical data load"

**Comment by denisekgosnell** [comment]
merging updates to IDM Match documentation

**Pull request merged**